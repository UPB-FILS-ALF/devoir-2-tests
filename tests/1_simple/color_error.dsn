& COLOR error

   COLOR:  color 200 0 30 
CIRCLE:   50 100  50     