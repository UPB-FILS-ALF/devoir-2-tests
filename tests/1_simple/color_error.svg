<?xml version="1.0" encoding="UTF-8" ?>
<svg xmlns="http://www.w3.org/2000/svg" version="1.1" width="1024" height="768">
	<circle cx="50" cy="100" r="50" stroke="rgb(0,0,0)"  stroke-width="1" fill="rgb(255,255,255)"/>
</svg>
