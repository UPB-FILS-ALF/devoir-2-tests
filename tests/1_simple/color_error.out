LINE 3 HAS ERROR: COLOR parameter 1 needs to be one of (pen, fill), you wrote color
